### ISSUE TITLE
Scores in mei viewer sometimes look cut

### ISSUE THREAD

**Issue by valeriyvan** [rism-digital/verovio.org#8]
Some combination of settings in mei viewer leads scores to look cut. But CLI verovio generates normal svg images for very same combination of parameters. It seems to be not verovio issue but javascript or browser environment issue.

![Bildschirmvideo aufnehmen 2020-07-03 um 22 33 57](https://user-images.githubusercontent.com/1630974/87666998-ec68b500-c769-11ea-8d4d-dd6ba21a9117.gif)

**Comment by lpugin** [comment]
Yes, I think so too. Probably because the score is overflowing the window and the size it is not reset when changing options.
